Vector Uninitialized in Calorimeter Plugin

The current `dev` as of ae101c078861a1fc576370893a25a682d72477b3 throws with GCC 5.3.0 on Piz Daint (with CUDA 8 backend)

```
/users/huebl/src/picongpu/include/pmacc/../pmacc/math/vector/Vector.hpp: In constructor 'picongpu::CalorimeterFunctor<CalorimeterCur>::CalorimeterFunctor(picongpu::float_X, picongpu::float_X, uint32_t, uint32_t, uint32_t, picongpu::float_X, picongpu::float_X, bool, picongpu::float3_X, picongpu::float3_X, picongpu::float3_X) [with CalorimeterCur = pmacc::cursor::BufferCursor<float, 3>; picongpu::float_X = float; uint32_t = unsigned int; picongpu::float3_X = pmacc::math::Vector<float, 3>]':
/users/huebl/src/picongpu/include/pmacc/../pmacc/math/vector/Vector.hpp:86:1: warning: '<anonymous>.pmacc::math::detail::Vector_components<long unsigned int, 2>::v[0]' is used uninitialized in this function [-Wuninitialized]
             dest[d] = src[d];
 ^
/users/huebl/src/picongpu/include/pmacc/../picongpu/plugins/particleCalorimeter/ParticleCalorimeterFunctors.hpp:65:389: note: '<anonymous>' was declared here
/users/huebl/src/picongpu/include/pmacc/../pmacc/math/vector/Vector.hpp:86:1: warning: '<anonymous>.pmacc::math::detail::Vector_components<long unsigned int, 2>::v[1]' is used uninitialized in this function [-Wuninitialized]
             dest[d] = src[d];
 ^
/users/huebl/src/picongpu/include/pmacc/../picongpu/plugins/particleCalorimeter/ParticleCalorimeterFunctors.hpp:65:389: note: '<anonymous>' was declared here
```

ccing @psychocoderHPC

--- a/include/picongpu/plugins/particleCalorimeter/ParticleCalorimeterFunctors.hpp
+++ b/include/picongpu/plugins/particleCalorimeter/ParticleCalorimeterFunctors.hpp
@@ -63,7 +63,7 @@ struct CalorimeterFunctor
                        const float3_X calorimeterFrameVecX,
                        const float3_X calorimeterFrameVecY,
                        const float3_X calorimeterFrameVecZ) :
-        calorimeterCur(nullptr, pmacc::math::Size_t<DIM2>()),
+        calorimeterCur(nullptr, pmacc::math::Size_t<DIM2>::create(0)),
         maxYaw(maxYaw),
         maxPitch(maxPitch),
         numBinsYaw(numBinsYaw),

